setup/docker/files/start.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Why do we need this file? Traccar creates log file automatically, so we don't need to create it. Basically we need to run java command only.